src/Airmap/QJsonWebToken/src/qjsonwebtoken.cpp

```diff
@@ -186,9 +186,9 @@ bool QJsonWebToken::setAlgorithmStr(QString strAlgorithm)
 	// set algorithm
 	m_strAlgorithm = strAlgorithm;
 	// modify header
-	m_jdocHeader = QJsonDocument::fromJson(QObject::trUtf8("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
+	m_jdocHeader = QJsonDocument::fromJson(QObject::tr("{\"typ\": \"JWT\", \"alg\" : \"").toUtf8()
```

Comment:
Is it ok to switch away from trUtf8 because there is no non-ascii in the strings?

Reply:
How would this look like?

I think using utf8 strings is fine. Calling tr is totally unnecessary in this context I guess.

Reply:
> Calling tr is totally unnecessary in this context I guess.

I believe that is correct. Can you just remove the 'tr' then?